src/web_app_skeleton/bin/setup

```diff
@@ -22,7 +22,7 @@ indent() {
 }
 
 echo "\n▸ Installing node dependencies"
-yarn install | indent
+yarn install --silent | indent
```

Comment:
Could you also echo "Done installing node dependencies | indent" so it isn't totally silent? Once the package.json warnings are fixed (#84) this will be extra important because it'll look like nothing happened if we don't print a success message:

Reply:
👍